packages/host/app/components/search-sheet/index.gts

```diff
@@ -208,7 +210,7 @@ export default class SearchSheet extends Component<Signature> {
   @cached
   get orderedRecentCards() {
     // Most recently added first
-    return [...this.operatorModeStateService.recentCards].reverse();
+    return [...this.recentCards.recentCards].reverse();
```

Comment:
feels a bit awkward to say `recentCards.recentCards`, maybe adjust to `recentCardsService.recentCards`?

Reply:
the challenge here is that the service file name is `recent-cards.ts`. to add the prefix we'd have to change the file name. but i think that your reasoning is why all the other services have the `-service` suffix.